src/components/notifications/AppSelector/index.tsx

```diff
@@ -100,52 +114,49 @@ const AppSelector: React.FC = () => {
-                        <div className="AppSelector__notifications-link">
-                          <img
-                            className="AppSelector__link-logo"
-                            src={
-                              app.metadata.icons?.length ? app.metadata.icons[0] : '/fallback.svg'
-                            }
-                            alt={`${app.metadata.name} logo`}
-                            onError={handleImageFallback}
-                            loading="lazy"
-                          />
-                          <div className="AppSelector__link__wrapper">
-                            <Text className="AppSelector__link__title" variant="small-500">
-                              {app.metadata.name}
-                            </Text>
-                            <Text className="AppSelector__link__subtitle" variant="small-500">
-                              {app.metadata.description}
-                            </Text>
-                          </div>
-                        </div>
+        <div className="AppSelector__wrapper">
+          <Label color="main">Subscribed</Label>
+          <ul className="AppSelector__list">
+            {loading
+              ? Array(3)
+                  .fill(
+                    <div className="AppSelector__link-item-skeleton">
+                      <div className="AppSelector__link-item-skeleton__icon"></div>
+                      <div className="AppSelector__link-item-skeleton__description"></div>
+                    </div>
+                  )
+                  .map(x => x)
+              : null}
+            {!loading &&
+              filteredApps?.map(app => (
+                <NavLink
+                  key={app.topic}
+                  to={`/notifications/${app.topic}`}
+                  className="AppSelector__link-item"
+                >
+                  <div className="AppSelector__notifications">
+                    <div className="AppSelector__notifications-link">
+                      <img
+                        className="AppSelector__link-logo"
+                        src={app.metadata.icons?.length ? app.metadata.icons[0] : '/fallback.svg'}
+                        alt={`${app.metadata.name} logo`}
+                        onError={handleImageFallback}
+                        loading="lazy"
+                      />
+                      <div className="AppSelector__link__wrapper">
+                        <Text className="AppSelector__link__title" variant="small-500">
+                          {app.metadata.name}
+                        </Text>
+                        <Text className="AppSelector__link__subtitle" variant="small-500">
+                          {app.metadata.description}
+                        </Text>
                       </div>
-                    </NavLink>
-                  </motion.div>
-                </AnimatePresence>
+                    </div>
+                  </div>
+                </NavLink>
```

Comment:
Why remove `AnimePresence` here?

Reply:
Main thing was not the removing animations, I should have been forgot